agent/engine/docker_task_engine.go

```diff
@@ -670,6 +670,10 @@ func (engine *DockerTaskEngine) pullContainer(task *apitask.Task, container *api
 	case apicontainer.ContainerCNIPause:
 		// ContainerCNIPause image are managed at startup
 		return dockerapi.DockerContainerMetadata{}
+
+	case apicontainer.ContainerNamespacePause:
```

Comment:
Wait, can you put this on one line?

`case api.ContainerCNIPause | api.ContainerNamespacePause:`

Reply:
done

Reply:
🤦 oops

use ',' not '|' .... '|' does entirely the wrong thing